.circleci/config.yml

```diff
@@ -0,0 +1,41 @@
+version: 2.1
+orbs:
+  ruby: circleci/ruby@0.1.2
+  browser-tools: circleci/browser-tools@1.0.0
+
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.5.1-stretch-node
```

Comment:
I think you can just add `-browsers` to this image name and skip the `install-chrome` lines below

Reply:
Played with that for a bit -- I ran into the problem with `-browsers`'s chrome version being about 10 versions out of date for selenium's chrome driver. I was able to reduce how much code I had in `rails_helper`, and was able to remove the `chromedriver` bit from this file, but I could not get a working build with taking out `install-chrome` as well.  Is that okay? I was really struggling to match up the version of chrome with the version needed by the driver.

Reply:
Yeah, this sector of testing has been in flux for a long time and there hasn't really emerged a stable and reliable way to get this working, unfortunately. So honestly if you have something that works locally and on Circle then that's probably fine. I'll pull it down and confirm it works on my machine as well.